README.md

```diff
@@ -389,94 +383,82 @@ function update_attributes($attributes) {
 ```
 
 
-
-Other SAML plugins
-------------------
+## Other SAML plugins
 
 The diversity and variable quality and features of SAML moodle plugins is a
 reflection of a great need for a solid SAML plugin, but the neglect to do
 it properly in core. SAML2 is by far the most robust and supported protocol
-across the internet and should be fully integrated into moodle core as both
+across the internet and should be fully integrated into Moodle core as both
 a Service Provider and as an Identity Provider, and without any external
 dependencies to manage.
 
 Here is a quick run down of the alternatives:
 
-### Moodle Core ###
+### Moodle Core
 
-* /auth/shibboleth - This requires a separately installed and configured
-  Shibbolleth install
+* `auth/shibboleth` - This requires a separately installed and configured
+  Shibboleth install.
 
-One big issue with this, and the category below, is as there is a whole extra
-application between moodle and the IdP, so the login and logout processes have
-more latency due to extra redirects. Latency on potentially slow mobile
-networks is by far the biggest bottle neck for login speed and the biggest
-complaint by end users in our experience.
+    One big issue with this, and the category below, is the extra
+    application between Moodle and the IdP, so the login and logout processes have
+    more latency due to extra redirects. Latency on potentially slow mobile
+    networks is by far the biggest bottleneck for login speed, and the biggest
+    complaint by end users in our experience.
 
-* /auth/oauth2
+* `auth/oauth2`
 
-OAuth2 has direct support in Moodle
+    OAuth2 has direct support in Moodle.
 
 
-### Plugins that require SimpleSamlPHP ###
+### Plugins that require SimpleSamlPHP
```

Comment:
Hi @danmarsden - what exactly do you mean by "links to auth_saml (not auth_saml2) have been removed accidentally"

Reply:
if you view the page in github:
https://github.com/catalyst/moodle-auth_saml2/blob/aa47f5ed3c1e9f474b155daad51428859f84395b/README.md

the first item under "Plugins that require SimpleSamlPHP" "auth_saml" is no longer clickable.

Reply:
Ah! Doh...